.eslintrc.cjs

```diff
@@ -614,13 +616,13 @@ const rules = {
     { ignorePrimitives: { boolean: true, string: true } }
   ],
   '@typescript-eslint/prefer-optional-chain': 1,
+  '@typescript-eslint/prefer-promise-reject-errors': [1, { allowEmptyReject: true }],
   '@typescript-eslint/prefer-readonly': 1,
   '@typescript-eslint/prefer-readonly-parameter-types': 0,
   '@typescript-eslint/prefer-reduce-type-parameter': 1,
   '@typescript-eslint/prefer-regexp-exec': 1,
   '@typescript-eslint/prefer-return-this-type': 0,
   '@typescript-eslint/prefer-string-starts-ends-with': 1,
-  '@typescript-eslint/prefer-ts-expect-error': 1,
```

Comment:
Replaced by https://typescript-eslint.io/rules/ban-tslint-comment/

Reply:
>This rule has been deprecated in favor of @typescript-eslint/ban-ts-comment.

Confused `ban-tslint-comment` and `ban-ts-comment`?

We don't use tslint, so maybe we can keep `ban-tslint-comment` disabled?